### PR TITLE
LPS-132349

### PR DESCRIPTION
**Pull request by rodrigopaulino**

### CODE REVIEW

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/test/js/Validation/Validation.es.js [hunk 1]
```diff
@@ -28,10 +28,12 @@ const defaultValue = {
...
 
-const ValidationWithProvider = (props) => (
-	<PageProvider value={{editingLanguageId: 'en_US'}}>
-		<Validation {...props} />
-	</PageProvider>
+const ValidationWithProvider = ({validations, ...props}) => (
+	<FormProvider initialState={{validations}}>
+		<PageProvider value={{editingLanguageId: 'en_US'}}>
+			<Validation {...props} />
+		</PageProvider>
```
Comment (miltonmc) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/test/js/Validation/Validation.es.js:35]: The page PageProvider is not being used by the `Validation` component, so you can just remove it
```suggestion
		<Validation {...props} />
```

**Review by miltonmc — CHANGES_REQUESTED** [review]
Good job!

Just a minor comment:

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/test/js/Validation/Validation.es.js (hunk 1 above)
Comment (rodrigopaulino) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/test/js/Validation/Validation.es.js]: Done. Thanks for your suggestion, @miltonmc

**Comment by rodrigopaulino** [comment]
ci:test:relevant

**Comment by rodrigopaulino** [comment]
ci:test:sf

**Review by miltonmc — APPROVED** [review]

**Comment by rodrigopaulino** [comment]
ci:forward

**Pull request closed without merging**